app/background.js

```diff
@@ -26,7 +26,11 @@ const checkToClose = function(cutOff: ?number) {
     if (!settings.get('paused')) {
 
       // Update the selected one to make sure it doesn't get closed.
-      chrome.tabs.query({active: true}, tabmanager.updateLastAccessed);
+      chrome.tabs.query({active: true, lastFocusedWindow: true}, tabmanager.updateLastAccessed);
+
+      if (settings.get('filterAudio') === true) {
+        chrome.tabs.query({audible: true}, tabmanager.updateLastAccessed);
```

Comment:
I looked at this again, and a clearer implementation for the user would be to filter them the same way pinned tabs are filtered.

The Tab Lock tab should show "Locked" in the right column rather than the countdown and give an explanation on hover that says something like "Tabs playing audio will not be closed".

Reply:
Thank you for your feedback. I have implemented the suggested changes in my most recent commit.